webserver/testing.py

```diff
@@ -1,14 +1,7 @@
-import flask_testing
 from db.testing import DatabaseTestCase
-from webserver import create_app
 
 
-class ServerTestCase(flask_testing.TestCase, DatabaseTestCase):
-
-    def create_app(self):
-        app = create_app()
-        app.config['TESTING'] = True
-        return app
+class ServerTestCase(DatabaseTestCase):
```

Comment:
I'm not sure about this file any more... The only thing that this gives us is the `temporary_login` method. We could just have a single base test class which we use for both database and application tests.

Reply:
I think application tests tend to use the database a lot anyways, so I agree. I was thinking of something like a `ListenBrainzTestCase` (or `AcousticBrainzTestCase` here) which would create a webapp and init the db etc. I'll open a ticket.

Reply:
[AB-311](https://tickets.metabrainz.org/browse/AB-311)